modules/va_forms/app/workers/va_forms/form_reloader.rb

```diff
@@ -72,6 +65,23 @@ def build_and_save_form(form)
       va_form
     end
 
+    def init_attributes(form)
+      mapped = {
+        title: form['fieldVaFormName'],
+        pages: form['fieldVaFormNumPages'],
+        language: form.dig('langcode', 'value'),
+        form_type: form['fieldVaFormType'],
+        form_usage: form.dig('fieldVaFormUsage', 'processed'),
+        form_tool_intro: form['fieldVaFormToolIntro'],
+        form_tool_url: form.dig('fieldVaFormToolUrl', 'uri'),
+        deleted_at: form.dig('fieldVaFormDeletedDate', 'value'),
+        related_forms: form['fieldVaFormRelatedForms'].map { |f| f.dig('entity', 'fieldVaFormNumber') },
+        benefit_categories: map_benefit_categories(form['fieldBenefitCategories'])
+      }
+      mapped[:form_details_url] = "#{FORM_BASE_URL}#{form.dig('entityUrl', 'path')}" if form['entityPublished']
+      mapped
```

Comment:
Robocop made me separate out this block from `build_and_save_form`. Something about too many lines in one function.

Reply:
Also, I used `FORM_BASE_URL`, but that constant is technically for the default domain where PDFs are stored, rather than where the form detail pages (HTML) live. Initially I thought about typing out `www.va.gov` again but decided to just reuse the var anyway.